Dockerfile

```diff
@@ -0,0 +1,59 @@
+# syntax = docker/dockerfile:1
```

Comment:
did this Dockerfile come from somwhere? are we planning to use Docker for local development?

Reply:
it comes by default by rails new app command. I don't think we need i'll remove iit.